src/Components/RecsListTable/RecsListTable.js

```diff
@@ -497,6 +512,7 @@ const RecsListTable = ({ query }) => {
         />
       )}
       <PrimaryToolbar
+        expandAll={{ isAllExpanded, onClick: collapseAll }}
```

Comment:
That is great that you used this simple property of the toolbar. However, I have spotted one bug. 

The `isAllExpanded` state parameter is not updated once you change the any of the filters/pagination. Imagine, that you have expanded all the rules, and if you navigate to the next page or apply any of the filters, you will see that the arrow button is still effective (= pointing downwards), but all the rows are hovered back.

I think, the only thing you need to enhance this `useEffect`: https://github.com/RedHatInsights/ocp-advisor-frontend/blob/c8fa96f39761ff0a05a93bfa0e3783a5a1ed2445/src/Components/RecsListTable/RecsListTable.js#L64

Reply:
Used terner operator in buildFilteredRows to have the same effect as you want
isOpen: isAllExpanded ? true : false